<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AliasManager</class>
 <widget class="QWidget" name="AliasManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tvAliasList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>281</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAdd">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>添加映射列表</string>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leAliasName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>别名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbChooseDirectory">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>选择目录</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDirectory">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>本机目录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbShare">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>分享链接</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
